### ISSUE TITLE
Code change delays

### ISSUE THREAD

**Issue by magbicaleman** [kalabox/kalabox#1488]
Bug Report
==========

- kalabox-v0.13.0-beta.2 on Mac 10.11.5 (El Capitan)

I'm not sure this qualifies as a bug, but I've noticed there is a significant time delay between editing a file and updating the code through the kalabox environment virtualization. 

I'm working with virtualizing Pantheon, so I'm not sure if this issue is specific to the environment. Any pointers on how to speed this up?

**Comment by pirog** [comment]
Hey @magbicaleman this is pretty extensively documented over here
http://docs.kalabox.io/en/stable/users/config/#sharing

If you are still having issues after reading the docs and optimizing your sync please reopen this

**Comment by TWGerard** [comment]
I am having this issue. I have read the documentation at the link you've provided and am not sure if I am just misunderstanding it or if it simply is not working for me. My `kalabox.yml` file has only one sharing directive: `share: 'data:/code'`, which is what I got when I cloned the site from Pantheon.

The severity of the delay varies greatly. At its best it is less than a second, but that is still too slow for me as I am using `livereload` and the file has not been updated on the Kalabox local site before `livereload` requests it. At its worst, such as tonight, my code changes have been ignored by Kalabox for hours despite restarting Kalabox, Docker, and the entire computer.
